tests/test.cpp

```diff
@@ -262,8 +262,8 @@ void AccessFlatBufferTest(const uint8_t *flatbuf, size_t length,
   auto vecofstructs = monster->testarrayofsortedstruct();
   if (vecofstructs) {  // not filled in monster_test.bfbs
     for (size_t i = 0; i < vecofstructs->size()-1; i++) {
```

Comment:
`vecofstructs->size()` already returns a `uoffset_t`. So if you just change the type of `i`, you don't need any casts.